core/templates/domain/admin/job-data-object.factory.ts

```diff
@@ -25,11 +25,14 @@ export interface JobDataBackendDict {
   'job_type': string;
   'status_code': string;
   'error': string;
-  'can_be_canceled'?: boolean;
   'is_cancelable': boolean;
   'id': string;
   'time_started_msec': number;
   'human_readable_time_started': string;
+  // This property is optional because this object factory is used for
+  // both 'recent_job_data' and 'unfinished_job_data' in admin page.
+  // 'recent_job_data' doesn't have this property.
+  'can_be_canceled'?: boolean;
```

Comment:
[Question unrelated to this PR] I see `is_cancelable` is that the same field?

Reply:
Nope! 

https://github.com/oppia/oppia/blob/6f99ce42489ff0d0cca419e750c4f0e3759b4bd3/core/controllers/admin.py#L89-L93